packages/docusaurus-theme-classic/src/theme/DocBreadcrumbs/index.tsx

```diff
@@ -22,11 +22,13 @@ function BreadcrumbsItemLink({
 }): JSX.Element {
   const className = clsx('breadcrumbs__link', styles.breadcrumbsItemLink);
   return href ? (
-    <Link className={className} href={href}>
-      {children}
+    <Link className={className} href={href} itemProp="item">
+      <span itemProp="name">{children}</span>
```

Comment:
Do we really need the extra span here and why?

Can't we apply "name" to the link directly?

Reply:
Yes, I've experimented. Without the span, the `name` is not correctly recognized.